src/schoology-extractor/tests/data_generation/users.py

```diff
@@ -0,0 +1,145 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+import logging
+from typing import Dict, List
+from http import HTTPStatus
+from faker import Faker
+from schoology_extractor.api.request_client import RequestClient
+
+fake = Faker("en_US")
+logger = logging.getLogger(__name__)
+
+VALID_SCHOOL_ID = "2908525646"
+VALID_ROLE_ID = "796380"
+VALID_EMAIL_DOMAIN = "studentgps.org"
```

Comment:
This makes sense. Also calls into question my comment above about making available to downstream users right in `schoology_extractor`. Separate directory is definitely better. I can live with the current directory structure, even though it  is a bit "odd".